Configuration/Skimming/python/PDWG_MuonPOGSkim_cff.py

```diff
@@ -0,0 +1,99 @@
+import FWCore.ParameterSet.Config as cms
+
+# Pick branches you want to keep
+MuonPOG_EventContent = cms.PSet(
+     outputCommands = cms.untracked.vstring(
+                     'drop *',
+                     'keep *_TriggerResults_*_*',
+                     'keep *_gmtStage2Digis_Muon_*',
+                     'keep *_offlineSlimmedPrimaryVertices_*_*',
+                     'keep *_offlineBeamSpot_*_*',                     
+                     'keep *_slimmedMETs_*_*',
+                     'keep *Muons*_slimmedMuons_*_*',
+                     'keep *_isolatedTracks_*_*',                   
+                     'keep *_slimmedPatTrigger_*_*',
+                     'keep *_slimmedJets_*_*',
+     )
+)
+
+
+import copy
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+from PhysicsTools.PatAlgos.producersLayer1.genericParticleProducer_cfi import patGenericParticles
+from PhysicsTools.PatAlgos.producersLayer1.muonProducer_cfi import patMuons
```

Comment:
why are the patGenericParticles and patMuons needed here?
please remove if not required